### DISCUSSION TITLE
Can't run on Android

### DISCUSSION THREAD

**Discussion by yfydodo939** [ORelio/Minecraft-Console-Client#2554] [Q&A] [answered]
I did everything as the tutorial but it keeps throwing errors below when I run it finally: 

GC heap initialization failed with error 0x8007000E
Failed to create CoreCLR, HRESULT: 0x8007000E

![Screenshot_20230731_084049](https://github.com/MCCTeam/Minecraft-Console-Client/assets/111677974/0212cd32-35c8-4352-9b4f-b4b2b242cc01)

**Comment by milutinke** [comment, marked as answer]
Execute the following commands, in order: 
`ulimit -v unlimited`
`sudo ulimit -u unlimited`
`sudo ulimit -s unlimited`
`sudo ulimit -n unlimited`
`sudo ulimit -l unlimited`

**Reply by yfydodo939** [reply]
Thank you very much! 😄
